task_reactredux/task1/src/containers/MenuContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import Menu from '../views/Menu/index';
+
+class MenuContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: false,
+      showMenu: true
+    }
+  }
+
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+  
+  componentWillMount() {
+    if (this.props.history.location.pathname === "/iTechArt-Lab-2018/about") {
```

Comment:
Please unify quotes in all your code. You should use either single or double quotes, without mixing